components/description-list/description-list-wrapper.js

```diff
@@ -0,0 +1,100 @@
+import { _generateBodyCompactStyles, _generateLabelStyles } from '../typography/styles.js';
+import { css, html, LitElement } from 'lit';
+import { classMap } from 'lit/directives/class-map.js';
+import ResizeObserver from 'resize-observer-polyfill/dist/ResizeObserver.es.js';
+
+export const descriptionListStyles = [
+	_generateLabelStyles('dt'),
+	_generateBodyCompactStyles('dd'),
+	css`
+		:host {
+			--d2l-dt-min-width: min-content;
+			--d2l-dt-max-width: 10rem;
+			--d2l-dd-min-width: 50%;
```

Comment:
Since these end up in the global namespace, I'd suggest prefixing them with `--d2l-description-list-*` just to avoid any future collisions or confusion.

Reply:
Or `d2l-dl-wrapper-*` I guess to match the name of the wrapper.

Reply:
Gah, thought I did this. Sorry